src/main/java/com/telq/sdk/clients/TelQTestClient.java

```diff
@@ -100,19 +125,45 @@ public static TelQTestClient getInstance() throws Exception {
     /**
      * Returns the list of all currently available networks.
      * @return List of {@link Network} which represent all currently available networks.
-     * @throws Exception
      */
-    public List<Network> getNetworks() throws Exception {
+    @Override
+    @SneakyThrows
+    public List<Network> getNetworks() {
         return apiConnectorService.getNetworks(authorizationService, new HttpGet(TelQUrls.getNetworksUrl()));
     }
 
+    @Override
+    @SneakyThrows
+    public List<Network> getNetworks(String mcc, String mnc) {
+        return getNetworks(mcc, mnc, null);
+    }
+
+    @Override
+    @SneakyThrows
+    public List<Network> getNetworks(String mcc, String mnc, String portedFromMnc) {
+        String url = TelQUrls.getNetworksUrl();
```

Comment:
Do you think it's much cleaner to use apache commons http client builder? We already uses this library.
```java
URIBuilder uriBuilder = new URIBuilder("http://baeldung.com/articles");
    uriBuilder.setPort(9090);
    uriBuilder.addParameter("topic", "java");
    uriBuilder.addParameter("version", "8");
    URL url = uriBuilder.build().toURL();
```

Reply:
Hi @tuvshuud ! Thank you for the comment, maybe we could ask @alxmann regarding this endpoint. He started initially working on the new API. I have only added the missing part for Session/Supplier API.